src/pages/wishListView/index.js

```diff
@@ -1,18 +1,20 @@
-import { List, ListItem, makeStyles } from '@material-ui/core'
 import React from 'react'
 import WishlistCard from '../../components/wishlistCard'
+import { List, ListItem, makeStyles } from '@material-ui/core'
 
 const useStyles = makeStyles(theme => ({
-  root: {},
+  ul: {
+    padding: "0 0 3rem 0"
+  },
```

Comment:
This will add unessential extra padding to all the wishlist element, and the problem is not kimited to wishlist but also other pages